butane_cli/src/main.rs

```diff
@@ -1,18 +1,26 @@
 use std::path::PathBuf;
 
-use clap::{Arg, ArgMatches};
+use clap::{value_parser, Arg, ArgMatches};
 
 use butane_cli::{
-    base_dir, clean, clear_data, collapse_migrations, delete_table, embed, handle_error,
-    list_migrations, migrate, Result,
+    clean, clear_data, collapse_migrations, delete_table, embed, handle_error, list_migrations,
+    migrate, working_dir_path, Result,
 };
 
 fn main() {
+    lazy_static::lazy_static! {
+        static ref WORKING_DIR_PATH: PathBuf = working_dir_path();
+    }
     let app = clap::Command::new("butane")
         .version(env!("CARGO_PKG_VERSION"))
         .author("James Oakley <james@electronstudio.org>")
         .about("Manages butane database migrations")
         .max_term_width(80)
+        .arg(
+            Arg::new("path").short('p').long("path")
+            .default_value(WORKING_DIR_PATH.as_os_str())
+            .value_parser(value_parser!(PathBuf))
```

Comment:
Could you add help text for this? Something like "select directory to look for butane state in"